LandSandBoat/server
# Usable Item Additions and Refinements

* Adds Scripts to make the following items usable:

Catholicon / +1
Clerics / Vicars Drink
Healing Mist / Powder
Mana Mist / Powder
Lucid Wings I / II
Sprinters Drink

* Migrates Effect Removal from individual Salve Scripts into item_utils
* Updates comments for a few item scripts
* Updates use time for a few items



**_I affirm:_**

## What does this pull request do?

Makes a number of items usable.

## Steps to test these changes

Login in a Test Character, add temp items, verify effects.
```
4146 -- Revit
4202 -- Daed wing
4206 -- catholicon
4208 -- cath +1
4254 -- Megalix
4255 -- mana powder
5322 -- healing powder
5395 -- clerics drink
5397 -- sprinters
5439 -- vicars
5440 -- Dusty Wing
5832 -- healing mist
5833 -- mana mist
5834 -- lucid wings i
5837 -- clear salve i
5838 -- clear salve ii
6475 -- lucid wings ii
```
